nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/CleanSurefireExecution.java

```diff
@@ -184,4 +186,15 @@ protected Xpp3Dom makeNode(String nodeName, String value) {
         node.setValue(value);
         return node;
     }
+
+    protected static String sanitizeAndRemoveEnvironmentVars(String toSanitize) {
```

Comment:
Add a comment for what this method does. Is it merely removing illegal characters from var names?